src/components/FormToggle.vue

```diff
@@ -76,6 +92,22 @@ export default {
             type: Number,
             default: 3,
         },
+        icon: {
+            type: String,
+            default: '',
+        },
+        iconSize: {
+            type: Number,
+            default: 12,
+        },
+        iconLeft: {
+            type: Boolean,
+            default: false,
+        },
+        iconRight: {
+            type: Boolean,
+            default: false,
+        },
```

Comment:
I think w do not need these props. We could just provide slots for left/right and if user wants an icon there he can edit it by himself. Now it is only <span>. What if user want there to be some more complicated element.
Imagine this case:
```
<form-toggle
 icon="icon-locked"
 icon-size="24"
 icon-left
 width="40"
 height="24"
 v-model="isChecked"
>
  <template #left>
    <div v-if="isChecked">
      ON
    </div>
  </template>
  <template #right>
    <div v-if="!isChecked">
      OFF
    </div>
  </template>
</form-toggle>
```